configs/src/deployments/json/zksyncSepolia.json

```diff
@@ -145,5 +145,162 @@
       "blockNumber": 2441217,
       "deploymentTx": "0x7bbeb9435090c339777a1f4bc62f891d2a6faa0ec331dd7a90f0800625d04a37"
     }
+  },
+  "v1.4.0": {
+    "ManagementDAOProxy": {
+      "address": "0xAEa0C1B96a4Cb95BCE15b98AFa87c32c915a3eC0",
+      "blockNumber": 2441177,
+      "deploymentTx": "0x4b7ded6dff5c99258f93733505122364acdc0878bd4b5e93d5f7878aec6d9655"
+    },
+    "ManagementDAOImplementation": {
+      "address": "0x39e836A6c32163733929B213965e3feC0007914a",
```

Comment:
Source not verified

Reply:
known issue, awaiting zkSync team to verify them